core/src/validator.rs

```diff
@@ -2765,8 +2782,10 @@ mod tests {
                 &cluster_info,
                 rpc_override_health_check.clone(),
                 &start_progress,
-            ),
-            Err(ValidatorError::NotEnoughLedgerData)
+            )
+            .unwrap_err()
+            .to_string(),
+            "Ledger does not have enough data to wait for supermajority"
```

Comment:
Can just matches!() on the validator error

Reply:
Done.